README.md

```diff
@@ -18,6 +18,18 @@ Create `.auth.json` in the directory you will run `bin/sync` from:
 }
 ```
 
+### GitHub Enterprise
+
+To connect to a GHE you will need to add the GHE host to the `.auth.json` file above:
+
+```
+{
+  "username": "github-username",
+  "password": "github-password",
```

Comment:
Would be better to use `token` for the GHE example, IMNSHO, since basic auth support is optional and token auth is generally preferred in all circumstances.